tools/twitter-api/scrolling.py

```diff
@@ -0,0 +1,91 @@
+from elasticsearch import Elasticsearch, helpers
+from random import randint
+import json
+config = json.load(open("config.json"))
+ 
+es = Elasticsearch(
+  [config["config"]["server"]],
+  # turn on SSL
+  use_ssl=True,
+  # no verify SSL certificates
+  verify_certs=False,
+  # don't show warnings about ssl certs verification
+  ssl_show_warn=False,
+  timeout=30,
```

Comment:
These should ideally be configurable. See other tools' config files in the repo. For example, use_ssl=True could break when running on a local machine.